SGMAcquaman_tests.pro

```diff
@@ -14,9 +14,9 @@ HEADERS += \
     source/tests/SGM/SGMEnergyPositionTest.h \
     source/tests/SGM/SGMEnergyControlTestView.h \
     source/tests/SGM/SGMGratingVelocityTests.h \
-    source/tests/SGM/SGMUndulatorVelocityTests.h \
-    source/tests/SGM/AMScalerAnalysisTest.h \
-    source/tests/SGM/AMScalerTestData.h
+    source/tests/SGM/SGMUndulatorVelocityTests.h #\
+    #source/tests/SGM/AMScalerAnalysisTest.h \
+    #source/tests/SGM/AMScalerTestData.h
 
```

Comment:
Yes, I have removed these files myself in #1691. It was the test mono
scaler data that is doing it. Now I made the test take input from a text
file.
On Nov 22, 2015 9:45 PM, "davidChevrier" <notifications@github.com> wrote:

> In SGMAcquaman_tests.pro
> <https://github.com/acquaman/acquaman/pull/1734#discussion_r45571149>:
>
> > @@ -14,9 +14,9 @@ HEADERS += \
> >      source/tests/SGM/SGMEnergyPositionTest.h \
> >      source/tests/SGM/SGMEnergyControlTestView.h \
> >      source/tests/SGM/SGMGratingVelocityTests.h \
> > -    source/tests/SGM/SGMUndulatorVelocityTests.h \
> > -    source/tests/SGM/AMScalerAnalysisTest.h \
> > -    source/tests/SGM/AMScalerTestData.h
> > +    source/tests/SGM/SGMUndulatorVelocityTests.h #\
> > +    #source/tests/SGM/AMScalerAnalysisTest.h \
> > +    #source/tests/SGM/AMScalerTestData.h
> >
>
> @iainworkman <https://github.com/iainworkman> and @dretrex
> <https://github.com/dretrex>
> I removed these because I noticed that a lot of builds really struggled on
> these files. I think they're really big (and fully of text?) ... I saw at
> least one case where the OPI hung up trying to do this compile.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/acquaman/acquaman/pull/1734/files#r45571149>.
>